provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitionerrole/PractitionerRoleTransformer.java

```diff
@@ -1,3 +1,94 @@
 package gov.va.api.health.providerdirectory.service.controller.practitionerrole;
 
-public class PractitionerRoleTransformer {}
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.allBlank;
+
+import gov.va.api.health.providerdirectory.api.datatypes.CodeableConcept;
+import gov.va.api.health.providerdirectory.api.datatypes.Coding;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.elements.Reference;
+import gov.va.api.health.providerdirectory.api.resources.PractitionerRole;
+import gov.va.api.health.providerdirectory.api.resources.PractitionerRole.PractitionerContactPoint;
+import gov.va.api.health.providerdirectory.service.PpmsPractitionerRole;
+import gov.va.api.health.providerdirectory.service.PpmsProviderSpecialtiesResponse;
+import gov.va.api.health.providerdirectory.service.ProviderContacts;
+import gov.va.api.health.providerdirectory.service.ProviderResponse;
+import gov.va.api.health.providerdirectory.service.controller.EnumSearcher;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PractitionerRoleTransformer implements PractitionerRoleController.Transformer {
+
+  @Value("${provider-directory.url")
+  String baseUrl;
+
+  @Override
+  public PractitionerRole apply(PpmsPractitionerRole ppms) {
+    // TODO organization reference is required
+    // location could be populated by caresites
+    ProviderResponse.Value provider = ppms.providerResponse().value().get(0);
+    return PractitionerRole.builder()
+        .resourceType("PractitionerRole")
+        .active(StringUtils.equalsIgnoreCase(provider.providerStatusReason(), "active"))
+        .practitioner(practitionerReference(provider))
+        .code(
+            CodeableConcept.builder()
+                .coding(codeCodings(ppms.providerSpecialtiesResponse()))
+                .build())
+        .telecom(
+            ppms.providerContacts().value().isEmpty()
+                ? telecoms(null)
+                : ppms.providerContacts()
+                    .value()
+                    .stream()
```

Comment:
This code can be simplified in it's own method, with the null check at the end. Something like:

```
telecoms = ppms.providerContacts.value().stream().flatMap(v -> telecoms(v).stream()).collect(Collectors.toList())
return telecoms.isEmpty() ? null : telecoms
```